Small improvements to Clock 3 and 9

waielal opened the pull request.

waielal committed:
Merge remote-tracking branch 'refs/remotes/CodingTrain/master'

waielal committed:
Small improvements to Clock 3

This should reduce all gaps.

waielal committed:
Clock 9 displays 12AM as 0 fixed

Oripy commented:
Good changes for clock 3. Since now there is a black circle drawn each loop, there is no need for black arcs.
I'm less sure about the hack to reduce the gap, it is less "elegant".
Thank you for the pull request!

waielal commented:
What du you mean by less elegant? This calculation `let padding = 0.1 * (7-i);` or the fact that/how I'm adding a padding to the arc?

waielal committed:
Little change to clock 3

Only add a constant padding when a cell has a right neighbour.
And fixed a little bug. Because of the break statement nothing was
display when hour() < 10.

Oripy commented:
@waielal the fact that we have to add some padding, it is a hack and not a true solution. But it's fine by me!
